src/district0x/ui/spec.cljs

```diff
@@ -5,7 +5,7 @@
 
 (s/def :db/load-node-addresses? boolean?)
 (s/def :db/web3 (constantly true)) ;; web3 is loaded asynchronously and is nil initially
-(s/def :db/node-url string?)
+(s/def :db/infura-id string?)
```

Comment:
Could you ask on discord if they're ok with only using Infura, i.e. they won't want to use their own Ethereum nodes?